ghtt/config.py

```diff
@@ -140,7 +143,37 @@ def get_repos(students: List[Person], mentors: Optional[List[Person]] = None) ->
         repo.organization = organization
         repo.mentors = [m for m in mentors if repo.group in m.groups]
         repos[repo.name] = repo
-    return repos
+
+    # Check if all repo's have expected number of students/mentors
+    ok_repos = {}
+    expected_group_count = get('expected_group_count', 1)
+    expected_mentor_count = get('expected_mentor_count', 0)
+    for repname, repo in repos.items():
+        count_expected = True
+        if expected_group_count and len(repo.students) != expected_group_count:
+            count_expected = False
+        if expected_mentor_count and len(repo.mentors) != expected_mentor_count:
+            count_expected = False
+        if count_expected:
+            ok_repos[repname] = repo
+        else:
+            if expected_mentor_count or repo.mentors:
+                click.secho("Group {} has {} students and {} mentors (expected {}/{}):"
+                            .format(repo.group, len(repo.students), len(repo.mentors),
+                                    expected_group_count, expected_mentor_count))
+            else:
+                click.secho("Group {} has {} students (expected {}):"
+                            .format(repo.group, len(repo.students), expected_group_count))
+            for stud in repo.students:
+                click.secho('   - student {} ({})'.format(stud.username, stud.comment))
+            for ment in repo.mentors:
+                click.secho('   - mentor {} ({})'.format(ment.username, ment.comment))
+            if click.confirm('Include {}'.format(repo.group), default=False):
```

Comment:
Wouldn't the `yes to all` options be useful here too? Maybe you can extract the `yes/no/stop [to all]` logic into a single function (`ask_to_skip`) that we can call in the `create_*` functions?

Example code in the `create_*` functions:

```python
pr_state = {}
size_state = {}

for repo in repos.values():
    
    if ask_to_skip("do you want to continue with creating PR for repo ...", pr_state):
        continue
    if (!repo.size_ok) && ask_to_skip("Group {} has {} students (expected {})", size_state):
        continue
    # create repo/pr/issue
```

What do you think?